src/hooks/api/events.js

```diff
@@ -0,0 +1,30 @@
+import { fetchWithRedirect } from '../../utils/fetchWithRedirect';
+import { useAuthenticatedQuery } from './useAuthenticatedQuery';
+
+const findToModerate = async (headers, searchQuery, start = 0, limit = 1) => {
+  const url = new URL(`${process.env.NEXT_PUBLIC_API_URL}/events/`);
+  url.search = new URLSearchParams({
+    q: searchQuery,
+    audienceType: 'everyone',
+    availableFrom: new Date().toISOString().split('.')[0] + 'Z',
```

Comment:
Nit: Maybe we could put this in a separate utils file like `fetchWithRedirect`, because most of our backend services can't deal with the microseconds in the `toISOString()` format right now so we'll probably need to do this formatting in multiple places.

Reply:
@bertramakers are you referring to this particular code? 
`.toISOString().split('.')[0] + 'Z'`